src/ui/Label/stringSet.ts

```diff
@@ -206,6 +208,9 @@ const stringSet = {
     DATE_FORMAT__MESSAGE_LIST__NOTIFICATION__UNREAD_SINCE: 'p \'on\' MMM dd',
     DATE_FORMAT__MESSAGE_LIST__DATE_SEPARATOR: 'MMMM dd, yyyy',
     DATE_FORMAT__THREAD_LIST__DATE_SEPARATOR: 'MMM dd, yyyy',
+    // File upload
+    FILE_UPLOAD_NOTIFICATION_COUNT_LIMIT: 'Up to %d files can be attached.',
+    FILE_UPLOAD_NOTIFICATION_SIZE_LIMIT: 'The maximum size per file is %d MB.',
```

Comment:
How about categorize with FILE_UPLOAD_NOTIFICATION?
```suggestion
    FILE_UPLOAD_NOTIFICATION__COUNT_LIMIT: 'Up to %d files can be attached.',
    FILE_UPLOAD_NOTIFICATION__SIZE_LIMIT: 'The maximum size per file is %d MB.',
```

Reply:
Thanks for the good suggestion!!